src/Sulu/Bundle/TrashBundle/Domain/Model/TrashItemInterface.php

```diff
@@ -0,0 +1,66 @@
+<?php
+
+/*
+ * This file is part of Sulu.
+ *
+ * (c) Sulu GmbH
+ *
+ * This source file is subject to the MIT license that is bundled
+ * with this source code in the file LICENSE.
+ */
+
+namespace Sulu\Bundle\TrashBundle\Domain\Model;
+
+use Sulu\Component\Security\Authentication\UserInterface;
+
+interface TrashItemInterface
```

Comment:
we could extend from the AuditableInterface so we don't need to handle the user our self

Reply:
Since trash items could theoretically also be stored in elasticsearch, the `AuditableInterface` wouldn't work anymore, because it's bound to doctrine

Reply:
And a `TrashItem` also can't have a `changer`

Reply:
as said in https://github.com/sulu/sulu/pull/6132/files#r701771320, it would be okay for me to not use the `AuditableInterface` because `changer` and `changed` are kind of confusing on a trash item